profile/README.md

```diff
@@ -1,14 +1,24 @@
 ## Artificial and Mechanical Intelligence
 
-Welcome to the GitHub organization of the Artificial and Mechanical Intelligence lab at the Italian Institute of Technology.
+Welcome to the GitHub organization of the Artificial and Mechanical Intelligence (AMI) lab at the Italian Institute of Technology.
 
 Our mission is to combine Artificial Intelligence with Mechanics to provide mankind the next generation of Humanoid Robots that will help us face the looming societal challenges.
 
 
 If you want to know more about us:
 * Visit our website at [**🌐 https://ami.iit.it/**](https://ami.iit.it/)
 * Follow us on Instagram https://www.instagram.com/artificialmechintelligence/
-* Follow us on Twitter https://twitter.com/ami_iit 
+* Follow us on Twitter https://twitter.com/ami_iit
+
+
+This GitHub organization contains different types of repositories:
+* **Reusable software** libraries that make algorithms developed in AMI, developed in C++, Python or MATLAB available to the research community and ready to be used. Examples: 
```

Comment:
Are you planning to complete this part with examples or is it a leftover?

Reply:
Leftover. I wanted to put this example, but then I tought people could ask me how I selected the examples I included. So I tought in a following PR I can just automatically create a list with the different repos (as done in https://github.com/robotology/.github/blob/main/.github/workflows/index-update.yml), and remove the examples here.